cli/src/infragpt/llm/auth.py

```
@@ -8,8 +8,8 @@
 from langchain_openai import ChatOpenAI
 from langchain_anthropic import ChatAnthropic
 
-from llm.models import MODEL_TYPE
-from llm.errors import AuthenticationError
+from infragpt.llm.models import MODEL_TYPE
+from infragpt.llm.errors import AuthenticationError
```

Ruff (0.11.9)

`infragpt.llm.errors.AuthenticationError` imported but unused

Remove unused import: `infragpt.llm.errors.AuthenticationError`

(F401)